Andrii_Fokin/1/Bob.py

```diff
@@ -0,0 +1,14 @@
+inStr = str(input())
```

Comment:
Naming conventions! Use in_str instead.
You don't need to convert input to str, because everything you get with input is str!